dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -245,4 +245,36 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
             LOG_IF_FAILED(::CoRevokeClassObject(static_cast<DWORD>(token.Cookie())));
         }
     }
+
+    bool PushNotificationManager::IsBIAvailable()
```

Comment:
BI?

Reply:
BI = Broker Infrastructure, we check if BI supports IBackgroundTaskBuilder5 but I can rename this to IsBackgroundTaskBuilder

Reply:
We are checking if we have support for IBackgroundTaskBuilder5 in the current build. Renamed to IsBackgroundTaskAvailable.